geode-core/src/main/java/org/apache/geode/internal/cache/BucketAdvisor.java

```diff
@@ -1670,7 +1663,7 @@ protected Set<BucketServerLocation66> getBucketServerLocations(int version) {
    *
    * @param id the member to use as primary for this bucket
    */
-  private void setPrimaryMember(InternalDistributedMember id) {
+  synchronized void setPrimaryMember(InternalDistributedMember id) {
```

Comment:
Why did we add synchronized here?

Reply:
was used for the test -- any caller should already get the synchronized lock.